### PR TITLE
Few fixes around remove liquidity and reset (liquidity/exchange) page

### PR DESCRIPTION
**Pull request by KarishmaBothara**
- After removing hardcoded core asset Id and fee asset Id from the reducer as the initial value, needed some fixes
- Remove liquidity few cases.

### CODE REVIEW

File: src/redux/reducers/ui/liquidity.reducer.ts [hunk 1]
```diff
@@ -168,7 +168,19 @@ export default handleActions<LiquidityState, any>(
...
             draft.exchangeRate = action.payload;
         }),
-        [LiquidityActions.LIQUIDITY_RESET]: () => initialState,
+        [LiquidityActions.LIQUIDITY_RESET]: produce((draft: LiquidityState) => {
+            draft.form.extrinsic = undefined;
+            draft.form.assetAmount = undefined;
+            draft.form.assetId = undefined;
+            draft.form.coreAmount = undefined;
+            draft.form.signingAccount = undefined;
+            draft.form.buffer = 0.05;
```
Comment (eupston) [inline src/redux/reducers/ui/liquidity.reducer.ts:177]: just out of curiosity what is this buffer used for?

**Review by eupston — APPROVED** [review]

File: src/redux/reducers/ui/liquidity.reducer.ts (hunk 1 above)
Comment (KarishmaBothara) [reply src/redux/reducers/ui/liquidity.reducer.ts:177]: slippage... when you add/remove liquidity or do exchange, there can be some change in the exchange/liquidity rate and so by default we keep the buffer as 0.05.. However, a user can change it as per need.

Comment (holygits) [inline src/redux/reducers/ui/liquidity.reducer.ts:177]: wasn't there a `const` for this somewhere...?

File: src/redux/epics/liquidity/liquidityRate.epic.ts [hunk 2]
```diff
@@ -107,8 +107,8 @@ export const getAssetLiquidityPriceEpic = (
                 return of(updateAsset1Amount(new Amount(assetAmount)));
             } else if (liquidityAction === REMOVE_LIQUIDITY) {
                 let liquidityAmount;
-                if (tradeAssetReserve.toString() === coreAssetReserve.toString()) {
-                    liquidityAmount = coreAmount.mul(totalLiquidity).div(coreAssetReserve);
+                if (totalLiquidity.toString() === coreAssetReserve.toString()) {
```
Comment (holygits) [inline src/redux/epics/liquidity/liquidityRate.epic.ts:110]: what does these changes do?

**Comment by holygits** [comment]
lgtm

File: src/redux/reducers/ui/liquidity.reducer.ts (hunk 1 above)
Comment (KarishmaBothara) [reply src/redux/reducers/ui/liquidity.reducer.ts:177]: ya.. can get it from window.config.FEE_BUFFER.... will update this

File: src/redux/epics/liquidity/liquidityRate.epic.ts (hunk 2 above)
Comment (KarishmaBothara) [reply src/redux/epics/liquidity/liquidityRate.epic.ts:110]: so I tried different scenarios to figure out this calculation works well... 
[scenarios.pdf](https://github.com/cennznet/cennzx-ui/files/6511602/scenarios.pdf)

**Pull request merged**